src/tests/ftest/pool/list_verbose.py

```diff
@@ -110,7 +110,8 @@ def create_expected(self, pool, scm_free, nvme_free, scm_imbalance,
                     "imbalance": nvme_imbalance
                 },
             ],
-            "mem_file_bytes": 0
+            "mem_file_bytes": scm_size if \
+            self.server_manager[0].manager.job.using_control_metadata else 0
```

Comment:
Shouldn't the line continuation be indented?
```suggestion
            "mem_file_bytes": scm_size if \
                self.server_manager[0].manager.job.using_control_metadata else 0
```

Reply:
Normally the GHA lint check would complain about this but looks like most GHA are deleted on this branch

Reply:
I tried that but the linter was giving me an error about dangling indent

Reply:
whoops, just copied and pasted with the typo, will correct